config/rbac/kustomization.yaml

```diff
@@ -18,9 +18,9 @@ resources:
 - topolvm_node_role.yaml
 - topolvm_node_role_bindings.yaml
 # topolvm-node scc
-- topolvm_node_scc.yaml
-- topolvm_node_scc_role.yaml
-- topolvm_node_scc_role_bindings.yaml
+#- topolvm_node_scc.yaml
+#- topolvm_node_scc_role.yaml
+#- topolvm_node_scc_role_bindings.yaml
```

Comment:
as the files are deleted, better remove these lines rather than commenting them?

Reply:
Ack. Missed this.